packages/shared/routes/setup/OnboardingNetwork.svelte

```diff
@@ -31,18 +31,23 @@
         <Text secondary classes="mb-8">{localize('views.network.body')}</Text>
     </div>
     <div slot="leftpane__action" class="flex flex-col space-y-4">
-        {#each Object.keys(NetworkType) as networkType}
+        {#each Object.keys(NetworkType) as networkType, index}
```

Comment:
If we use `Object.values(NetworkType)` we could avoid using `index === 0 ? this : that` but instead check if its `Mainnet`, eg:

```js
{#each Object.values(NetworkType) as networkType}
    <Button
        icon={networkIcon[networkType]}
        iconColor={networkType === NetworkType.Mainnet ? `${$newProfile?.networkProtocol}-highlight` : 'blue-500'}
        classes="w-full"
        secondary
        disabled={!featureFlags?.onboarding?.[$newProfile?.networkProtocol]?.[networkType]?.enabled}
        onClick={() => onClick(networkType)}
    >
        {localize(`views.network.${$newProfile?.networkProtocol}.${networkType}.title`)}
        {#if !$mobile}
            <Text secondary smaller>
                {localize(`views.network.${$newProfile?.networkProtocol}.${networkType}.body`)}
            </Text>
        {/if}
    </Button>
{/each}
```
wdyt?

Reply:
I've been thinking of changing the Object.keys to Object.values the more I go into this file, so I went ahead and made the change.